### DISCUSSION TITLE
esp32 wifi and ethernet webserver?

### DISCUSSION THREAD

**Discussion by elektroboard** [khoih-prog/WebServer_ESP32_W5500#3] [Q&A]
Hello.
For a project, I want to develop a webserver application over both esp32's wifi and w5500 ethernet module.

But I couldn't do both together. What do you recommend.

Thanks.

**Comment by khoih-prog** [comment]
Check [**WiFi and Ethernet Bridge?** #26](https://github.com/khoih-prog/EthernetWebServer/issues/26#issuecomment-1014782394) and adapt your core
